corehq/apps/auditcare/tests/test_tasks.py

```diff
@@ -0,0 +1,93 @@
+from datetime import datetime
+
+from ..models import AccessAudit, NavigationEventAudit
+from ..tasks import copy_events_to_sql
+from .testutils import AuditcareTest, delete_couch_docs, save_couch_doc
+
+
+class TestCopyEventsToSQL(AuditcareTest):
```

Comment:
Doesn't look like there is any test here verifying the Couch query limit/batching actually works. I'm thinking of a test that uses a batch size of 1 or 2 (not sure if 1 will work), and verifying that the entire date range, which consists of multiple batches, is copied.

Reply:
I will add them.